driver/pace/driver/configs/comm.py

```diff
@@ -0,0 +1,110 @@
+import abc
```

Comment:
I'd like to move more code out of `driver.run` and into namespaces, but left this for another PR.